scheduler/docs/configuration.adoc

```diff
@@ -375,6 +376,11 @@ The configuration values are defined as follows:
   An integer representing the number of millisecond intervals at which progress updates will be published to datomic.
   The default value is 2500.
 
+`:sequence-cache-threshold`::
+  An integer representing the size of the task id to progress sequence cache to maintain.
```

Comment:
The max number of items in the task sequence cache.

Reply:
Done.